### ISSUE TITLE
"SELECT SINGLE" + "ORDER BY", expect error

### ISSUE THREAD

**Issue by smardcode** [abaplint/abaplint#1751]
The following statement is not valid, but the linter does not complain:
```abap
  SELECT SINGLE *
    FROM vbak
    INTO @DATA(ls_vbak)
    ORDER BY vbeln.
```

**Comment by larshp** [comment]
yea, there are some gaps, however I prioritize getting the allowed language right first, then cutting down on the loopholes

**Comment by smardcode** [comment]
sure.

**Comment by larshp** [comment]
@smardcode try again

https://syntax.abaplint.org/?filter=select#/abap/expression/Select

![image](https://user-images.githubusercontent.com/5888506/234854662-36188fa7-a3a8-4e2d-a0c5-1d0298171e98.png)

**Comment by larshp** [comment]
closing
